src/banana/data/genpdf/cli.py

```diff
@@ -0,0 +1,26 @@
+import click
```

Comment:
Add a module docstring here: "This module defines the CLI - see here_point_to_cli_description"

Reply:
How can I put the link? I tried but I can't find the right sintax...

Reply:
see e.g. here: https://github.com/N3PDF/banana/blob/main/docs/source/index.rst